scheduler/pkg/kafka/topics_test.go

```diff
@@ -39,9 +39,23 @@ func TestGetModelTopic(t *testing.T) {
 		{
 			name:          "model",
 			pipelineName:  "test",
-			topicNamer:    NewTopicNamer("default", "foo"),
+			topicNamer:    NewTopicNamer("default", ""),
 			stepReference: "model1.inputs",
-			expected:      "foo.default.model.model1.inputs",
+			expected:      "seldon.default.model.model1.inputs",
+		},
+		{
+			name:          "model with custom prefix",
+			pipelineName:  "test",
+			topicNamer:    NewTopicNamer("default", "foo.bar"),
+			stepReference: "model1.inputs",
+			expected:      "foo.bar.default.model.model1.inputs",
+		},
+		{
+			name:          "model with custom prefix with a dot",
+			pipelineName:  "test",
+			topicNamer:    NewTopicNamer("default", "foo.bar."),
```

Comment:
nit: it is perhaps from a user experience point of view that is the prefix is set to `foo.bar.` then it is likely that the user is not requiring an extra dot in the full topic name.

Reply:
It seemed simpler not to even check and allow this.

Reply:
actions:
 - Clarify valid characters for Kafka topic
 - Update docs to suggest not having dot at end